Makefile

```diff
@@ -3,7 +3,7 @@ test:
 testcov:
 	py.test --cov=ideascube/ --cov-report=term-missing --migrations
 develop:
-	pip3 install -r requirements-dev.txt
+	pip3 install --no-use-wheel -r requirements-dev.txt
```

Comment:
We can't do anything here unfortunately.

Older pip (as on Debian Jessie) uses the `--no-use-wheel` argument.

Newer pip (as on our Fedora workstations, or on Debian Stretch) use `--no-binary :all:`.

We can't move to the new one or else `make develop` won't work on Debian Jessie any more.

Right now I'm just hoping that we'll move to Stretch before pip upstream deletes this flag.

Checking the version of pip in the Makefile to use the right option would be pretty painful to implement (because Makefile targets aren't a shell script) so for now I'd rather just punt on the issue, and do it if it ever becomes a problem.